src/components/Description.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'preact/hooks'
+import AvailableStepFlow from 'models/AvailableStepFlow'
+import Messages from 'models/Messages'
+import classnames, {
+  fontSize,
+  textAlign,
+  textColor,
+  textDecoration,
+  touchAction,
+  userSelect,
+} from 'classnames/tailwind'
+import postWebViewMessage from 'helpers/postWebViewMessage'
+
+const description = classnames(fontSize('text-sm'), textAlign('text-center'))
+const descriptionLink = (active?: boolean) =>
+  classnames(
+    description,
+    touchAction('touch-none'),
+    userSelect('select-none'),
+    textColor(active ? 'text-link-hover' : 'text-link'),
```

Comment:
For styling we can use built-in tailwind stuff like `hover:text-link-hover`, why not?

Reply:
Cause it doesn't work properly, it doesn't handle touch events and change the styles only on long press